src/ur5e_arm.cpp

```diff
@@ -267,11 +282,13 @@ void UR5eArm::move(std::vector<Eigen::VectorXd> waypoints) {
     segments.push_back(waypoints.size() - 1);
 
     // set velocity/acceleration constraints
-    BOOST_LOG_TRIVIAL(debug) << "generating trajectory with max speed: " << speed * (180.0 / M_PI) << std::endl;
+    const double move_speed = speed.load();
+    const double move_acceleration = acceleration.load();
+    BOOST_LOG_TRIVIAL(debug) << "generating trajectory with max speed: " << move_speed * (180.0 / M_PI) << std::endl;
     Eigen::VectorXd max_acceleration(6);
     Eigen::VectorXd max_velocity(6);
-    max_acceleration << acceleration, acceleration, acceleration, acceleration, acceleration, acceleration;
-    max_velocity << speed, speed, speed, speed, speed, speed;
+    max_acceleration << move_acceleration, move_acceleration, move_acceleration, move_acceleration, move_acceleration, move_acceleration;
+    max_velocity << move_speed, move_speed, move_speed, move_speed, move_speed, move_speed;
```

Comment:
currently we still set every join to the same max vel/acc. shouldn't be too much of a lift to separate these out if requested

Reply:
I think no need to do these now we don't know how to set them well anyway